src/hooks/useConcepts.ts

```diff
@@ -1,53 +1,58 @@
 import { useMemo } from 'react';
-import useSWRInfinite from 'swr/infinite';
-import { type FetchResponse, openmrsFetch, type OpenmrsResource, restBaseUrl } from '@openmrs/esm-framework';
-import { useRestApiMaxResults } from './useRestApiMaxResults';
-
-type ConceptFetchResponse = FetchResponse<{ results: Array<OpenmrsResource> }>;
+import useSWR from 'swr';
+import { type Concept, type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
 
 const conceptRepresentation =
   'custom:(uuid,display,conceptClass:(uuid,display),answers:(uuid,display),conceptMappings:(conceptReferenceTerm:(conceptSource:(name),code)))';
 
-export function useConcepts(references: Set<string>): {
-  concepts: Array<OpenmrsResource> | undefined;
+type ConceptRepresentation = Pick<Concept, 'uuid' | 'display'> & {
+  conceptClass: Pick<Concept['conceptClass'], 'uuid' | 'display'>;
+  answers: {
+    uuid: string;
+    display: string;
+  };
+  conceptMappings: {
+    conceptReferenceTerm: {
+      conceptSource: {
+        name: string;
+      };
+      code: string;
+    };
+  };
```

Comment:
```suggestion
  answers: Array<{
    uuid: string;
    display: string;
  }>;
  conceptMappings: Array<{
    conceptReferenceTerm: {
      conceptSource: {
        name: string;
      };
      code: string;
    };
  }>;
```

Reply:
Good points. In my ideal world, these would actually be defined upstream, but the definitions for them in esm-core are currently `any`.